test/module_6/page/app.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-undef */
+const { browser, element, by, $$ } = require("protractor");
+
+class DemoApp {
+    constructor() {
+        this.firstNumber = element(by.model("first"));
+        this.secondNumber = element(by.model("second"));
+        this.goButton = element(by.id("gobutton"));
+        this.operator = element(by.model("operator"));
+        this.history = $$("tr.ng-scope");
+        this.text = $("div:nth-child(1) > h3:nth-child(1)");
+        
+    }
+
+    get() {
+        browser.get("http://juliemr.github.io/protractor-demo/");
+    }
+
+    getTitle() {
+        browser.executeScript("document.querySelector('body').style.background = 'grey'");
```

Comment:
for future: I would recommend using async/await or promise chaining for async functions. I wouldn't hope on control flow.

Reply:
ok)